script/dev/start-server.sh

```diff
@@ -13,14 +13,6 @@ if [[ "$PWD" != "$HOME"* ]]; then
   exit 1;
 fi
 
-ORCH_URL_ROOT=${ORCH_URL_ROOT:-'https://firecloud-orchestration.dsde-dev.broadinstitute.org'}
+docker run --rm -it -v "$PWD":/working broadinstitute/dsde-toolbox render-templates.sh local $VAULT_TOKEN
```

Comment:
Add `src/docker` to the end of this so we can use paths that make more sense for this project.